README.adoc

```diff
@@ -90,14 +90,31 @@ Build the example located in the _upstream-1_ folder:
     git commit -a -m"Pages update"
     git push origin +gh-pages
     
-=== Creating References to Anchors
+=== Creating Anchors and xrefs
 
+==== Dynamic Anchor and Dynamic xref
 An anchor (for example: [[Testing the App]]) cannot appear more than once in an *.adoc* file.
 As Modular Doc files are regularly reused in the same *.adoc* file, this results in a build error.
 See link:http://asciidoctor.org/docs/user-manual/#include-multiple[Including Multiple Anchors] to negate this issue.
 
 To best understand the information above, see the example located in the repo.
 
+==== Dynamic Anchors and Hard-Coded xref
```

Comment:
Basically, what was pushed in (https://github.com/feedhenry-raincatcher/raincatcher-docs/pull/20) is the latest and greatest.

To save every bodies time right now is this:

1) copy all content from "README.adoc" in (https://github.com/feedhenry-raincatcher/raincatcher-docs/pull/20)  

2)and replace it with content in this PR readme.

Job done. Again, sorry for this situation, they will become will become non existent as now that I know how the flow (upstream and forks) works